tests/test__collection_api.py

```diff
@@ -377,6 +381,10 @@ def test__find_removed_and_changed_options(self):
             with self.assertRaises(TypeError):
                 self.db.collection.find({}, **option)
 
+    def test__find_collation_option(self):
```

Comment:
I think you can drop this test, as the collation param was added to find while your PR was waiting for a review.